packages/abi-gen/.gitignore

```diff
@@ -0,0 +1,3 @@
+test-cli/artifacts
+test-cli/python/generated-wrappers
+test-cli/typescript/generated-wrappers
```

Comment:
Why not put this stuff in the top-level `.gitignore`? (with the `/packages/abi-gen/` prefix of course).

Reply:
I initially thought it was easier to track it here since these are all specific to this project only. At this point though the `output` vs `expected-output` idea makes a lot of sense, and that would also make sense to add to the top-level gitignore to start a convention of CLI test folder names.

Changing.